src/codeu/chat/server/Model.java

```diff
@@ -71,6 +72,28 @@ public int compare(Time a, Time b) {
   public void add(User user) {
     currentUserGeneration = userGenerations.make();
 
+    /// SQLite
+    Connection c = null; /// Create connection
+    Statement stmt = null;
+    try {
+      Class.forName("org.sqlite.JDBC");
+      c = DriverManager.getConnection("jdbc:sqlite:test.db"); /// Connect to database
+      c.setAutoCommit(false); 
+      System.out.println("Opened database successfully model");
+
+      stmt = c.createStatement();
+      ///Test data to be entered into table
+      String sql = "INSERT INTO USER (ID,TIME,NAME) " +  
+                   "VALUES (1234, 252525, 'Papa');"; 
+      stmt.executeUpdate(sql);
+      stmt.close();
+      c.commit();
+      c.close();
+    } catch ( Exception e ) {
+      System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+      System.exit(0);
+    }
+    System.out.println("Records created successfully");
     userById.insert(user.id, user);
     userByTime.insert(user.creation, user);
     userByText.insert(user.name, user);
```

Comment:
After thinking about this some more, it may make more sense for this logic to exist on the Server rather than the client. I would move all of the database logic to the Model.java class as it normally handles all database operations. Then you could define an "initialize()" method (or simply add logic to the constructor) on the Model class that does this setup, of either create the database if it does not exist, or reading the users from the database if it does. I believe the client and server already are hooked up to call the model methods for add/get(messages) simply adding the initialize method on the Model so that it reads the initial messages and users from the database would work.